### ISSUE TITLE
Training loss does not decrease and test accuracy always is 0.5

### ISSUE THREAD

**Issue by a-marchesi** [BVLC/caffe#4427]
Hi everyone, 
I'm using Caffe for a classification task, but I'm having several problems in training my CNN. 
More in detail, after some iterations, the loss takes always the same value even if I try different values for the parameters defined in the solver. 
Moreover, in the last experiments it seems to be almost constant around the value 0.69 (attached a plot of the loss during training)
![provastepshuffleok](https://cloud.githubusercontent.com/assets/19926949/16655102/1e15ba1e-4459-11e6-9e00-2b71d0b63586.png)
The other issue is related to the accuracy on the test set. So far in all the experiments that I’ve carried out the accuracy on the test set is always 0.5. I think that this fact with the results I have on the loss mean that the CNN is not learning during training. 
Does anyone have any suggestion to better understand the reason of these problems?

**Comment by seanbell** [comment]
Please ask questions about modeling/usage on the mailing list, thanks!

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
